concourse/pipelines/fast-startup-and-shutdown-cf-env.yml

```diff
@@ -21,6 +21,11 @@ resource_types:
     repository: ghcr.io/alphagov/paas/semver-resource
     tag: f2a80c95481056aa57d837e3b14f0012b542fdb3
 
+- name: keyval
+  type: registry-image
+  source:
+    repository: ghcr.io/cludden/concourse-keyval-resource
```

Comment:
We seem to have a convention to put all our concourse resources into containers we control.

Reply:
fixed